src/app/server/Server.h

```diff
@@ -233,7 +237,7 @@ struct CommonCaseDeviceServerInitParams : public ServerInitParams
         {
             // WARNING: PersistentStorageOperationalKeystore::Finish() is never called. It's fine for
             //          for examples and for now.
-            ReturnErrorOnFailure(sPersistentStorageOperationalKeystore.Init(this->persistentStorageDelegate));
+            // ReturnErrorOnFailure(sPersistentStorageOperationalKeystore.Init(this->persistentStorageDelegate));
```

Comment:
should this be removed or have the ```#if !CHIP_CRYPTO_PSA``` guard around it?

Reply:
Should be guarded, thanks for spotting, must have missed that one!